### PR TITLE
Frontend Test Cases for Login/Register

### PR DESCRIPTION
**Pull request by rekandiah**
- Removed unnecessary files from chutzpah

- Used Selenium IDE plugin to run testcases for login and register on CUSE website

- Exported test cases  generated by Selenium as C# files 

- Need to add the necessary dependencies such that the frontend tests can be run within Visual Studio

### CODE REVIEW

File: chutzpah/Samples/readme.txt [hunk 1]
```diff
@@ -1,26 +0,0 @@
-Chutzpah Samples
```
Comment (coreylevine) [inline chutzpah/Samples/readme.txt]: I think we should keep this file. It shows some of the useful aspects of QUnit and Chutzpah

Comment (rekandiah) [inline chutzpah/Samples/readme.txt]: Yeah you're right! I'll make a copy of this file so we use for future reference.

Comment (coreylevine) [inline chutzpah/Samples/readme.txt]: Good idea. I will approve the pull request now.

**Comment by coreylevine** [comment]
#27 
 - Unnecessary files were removed
 - Frontend tests were added for login page
 - Frontend tests were added for register page

**Pull request merged**